src/components/common/HelpSection.vue

```diff
@@ -1,32 +1,26 @@
 <template>
-  <div v-if="helpSection">
```

Comment:
This component shouldn't decide whether to display itself; the parent should decide.